docs/guides/configure_service_blockchain.md

```diff
@@ -0,0 +1,201 @@
+The {{open_autonomy}} framework supports running agent services in different chains
+by defining certain configuration parameters appropriately.
+
+For example, if you are developing a service to be run in Ethereum, you might first want to use
+a development environment like [Hardhat](https://hardhat.org/) or [Ganache](https://trufflesuite.com/ganache/) as your testbed, then migrate to a public testnet like [Ropsten](https://ropsten.etherscan.io/) or [Görli](https://goerli.net/), and ultimately run your service in the Ethereum main chain.
+
+To use a service in a particular chain, you must ensure that the the relevant agent addresses are funded in that chain.
+
+## What will you learn
+
+In this guide, you will learn how to:
+
+  * Configure an agent service to be run in a particular chain.
+  * Configure the test classes appropriately.
+
+
+## Configuring the agent service
+To configure an agent service to work with a particular chain, you must configure the parameters `address` and `chain_id` accordingly in the connection `valory/ledger`. This configuration can be set up by overriding the `valory/ledger` configuration at agent level or at [service level](./service_configuration_file.md#service-level-overrides).
+
+
+### Agent-level override
+At agent level, the agent configuration file `aea-config.yaml` should contain an override for the parameters of the connection `valory/ledger`, either using hardcoded values or environment variables as follows:
+
+=== "Using hardcoded values"
+
+    ```yaml
+    (...)
+    ---
+    public_id: valory/ledger:0.19.0
+    type: connection
+    config:
+      ledger_apis:
+        ethereum:
+          address: http://host.docker.internal:8545
+          chain_id: 31337
+          <other_params>
+    ```
+
+=== "Using environment variables"
+
+    ```yaml
+    (...)
+    ---
+    public_id: valory/ledger:0.19.0
+    type: connection
+    config:
+      ledger_apis:
+        ethereum:
+          address: ${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_ADDRESS:str:http://localhost:8545}
+          chain_id: ${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_CHAIN_ID:int:31337}
+          <other_params>
+    ```
+
```

Comment:
fyi the long names are no longer to be required by the service overrides, so for example 
```
${CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_ADDRESS:str:http://localhost:8545}
```
can be replaced with:
```
${str:http://localhost:8545}
```

Reply:
We just have to define the type, the default value, and maintain the `${}`.

Reply:
Fixed.